hydromt_wflow/utils.py

```diff
@@ -3,26 +3,33 @@
 import logging
 from os.path import abspath, dirname, join
 from pathlib import Path
-from typing import Any, Callable, Dict, Union
+from typing import Any, Callable, Dict, Union, cast
 
 import geopandas as gpd
 import numpy as np
-import tomlkit
 import xarray as xr
 from hydromt.io import open_timeseries_from_table
 from hydromt.vector import GeoDataArray
 from hydromt.workflows.grid import grid_from_constant
-from tomlkit.items import Key
+
+from .naming import (
+    WFLOW_NAMES,
+    WFLOW_SEDIMENT_NAMES,
+    WFLOW_SEDIMENT_STATES_NAMES,
+    WFLOW_STATES_NAMES,
+)
 
 logger = logging.getLogger(__name__)
 
-DATADIR = Path(join(dirname(abspath(__file__)), "data"))
+DATADIR = join(dirname(abspath(__file__)), "data")
 
 __all__ = [
-    "get_config",
-    "get_grid_from_config",
     "read_csv_results",
+    "get_config",
     "set_config",
+    "get_grid_from_config",
+    "convert_to_wflow_v1_sbm",
+    "convert_to_wflow_v1_sediment",
```

Comment:
This two functions, as well as all functions that are about wflow version conversion have been moved to version_upgrade.py script. It seems they are now duplicated?